deployment/live/aws/test/README.md

```diff
@@ -19,6 +19,7 @@ define a test environment to run TesseraCT, backed by Trillian Tessera.
 At a high level, this environment consists of:
 - One RDS Aurora MySQL database
 - One S3 Bucket
+- Two secrets in AWS Secrets Manager
```

Comment:
I find it misleading that one of the AWS Secrets is not really a secret: it's a public key. Maybe make it explicit by stating that a log public and private key are stored in Secrets Manager?

Reply:
The "secrets" here is the terminology in [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/). It could be confusing to those who are not familiar with AWS. I've added more information (i.e. the log public/private keys) to the secrets.